servers.json

```diff
@@ -1338,5 +1338,12 @@
         "addresses": [
             "tearfulmc.com"
         ]
+    },
+    {
+        "id": "magiccraft",
+        "name": "MagicCraft",
+        "addresses": [
+            "mc.magiccraft.fr"
```

Comment:
Lunar Client services automatically detect sub-domains. Please remove the `mc.` prefix.